public/content/lessons/2016/cross-products/index.mdx

```diff
@@ -1,10 +1,253 @@
+
+Here, I'll try to hit the main points that students are usually shown about the cross product, and in the next chapter I'll show a view which is less commonly taught, but really satisfying when you learn it.
+
+## Two dimensions
+
+We'll start in two dimensions. If you have two vectors, $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, think about the parallelogram that they span out. What I mean by this is that if you take a copy of $\vec{\mathbf{v}}$, and move its tail to the tip of w, and you take a copy of $\vec{\mathbf{w}}$, and move its tail to the tip of $\vec{\mathbf{v}}$, the four vectors enclose a certain parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogram.svg" 
+/>
+
+The cross product of $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, written with the x-shaped multiplication symbol ($\times$), is the area of this parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramFormula.svg" 
+/>
+
+Well, almost, we also need to consider orientation. Basically, if $\vec{\mathbf{v}}$ is on the right of $\vec{\mathbf{w}}$, then $\vec{\mathbf{v}}$ cross $\vec{\mathbf{w}}$ is positive and equal to the area of that parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramPositive.svg" 
+/>
+
+But if $\vec{\mathbf{v}}$ is on the left of $\vec{\mathbf{w}}$, the cross product is negative 1 times the area of that parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramNegative.svg" 
+/>
+
+Notice, this means order matters. If you swap $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, and instead consider $\vec{\mathbf{w}}$ cross $\vec{\mathbf{v}}$, the cross product becomes the negative of whatever it was before.
+
+$$
+\vec{\mathbf{v}} \times \vec{\mathbf{w}} = -\vec{\mathbf{w}} \times \vec{\mathbf{v}}
+$$
+
+The way I always remember this is that when you take the cross product of the two basis vectors in order, i hat cross j hat, the result should be positive. In fact, the order of your basis vectors is what defines orientation. So since i hat is on the right of j hat, I remember that v cross w is positive whenever w is on the right of v.
+
+<Figure
+    image="./figures/two-dimensions/BasisVectorsSign.svg" 
+/>
+
+So for example, with the vectors shown here, I'll just tell you that the area of that parallelogram is 7. And since v is on the left of w, the cross product should be negative, so v cross w is -7.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramCalculation.svg" 
+/>
+
+**Given the diagram below:**
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramQuestion1.svg" 
+/>
+
+<Question
+  question="Is the cross product $\vec{\mathbf{v}} \times \vec{\mathbf{w}}$ positive?"
+  choice1="true"
+  choice2="false"
+  answer={1}
+>
+
+The cross product $\vec{\mathbf{v}} \times \vec{\mathbf{w}}$ is positive since $\vec{\mathbf{v}}$ is on the right of $\vec{\mathbf{w}}$.
```

Comment:
```suggestion
The cross product $\vec{\mathbf{v}} \times \vec{\mathbf{w}}$ is positive since $\vec{\mathbf{w}}$ is a counterclockwise rotation away from $\vec{\mathbf{v}}$.
```